src/libstd/lib.rs

```diff
@@ -302,9 +302,10 @@
 #![feature(non_exhaustive)]
 #![feature(alloc_layout_extra)]
 #![feature(maybe_uninit)]
-#![cfg_attr(target_env = "sgx", feature(global_asm, range_contains, slice_index_methods,
-                                        decl_macro, coerce_unsized, sgx_platform,
-                                        min_const_unsafe_fn))]
+#![cfg_attr(all(target_vendor = "fortanix", target_env = "sgx"),
+            feature(global_asm, range_contains, slice_index_methods,
+                    decl_macro, coerce_unsized, sgx_platform,
+                    min_const_unsafe_fn))]
```

Comment:
```
warning: the feature `min_const_unsafe_fn` has been stable since 1.33.0 and no longer requires an attribute to enable
   --> src/libstd/lib.rs:308:21
    |
308 |                     min_const_unsafe_fn))]
    |                     ^^^^^^^^^^^^^^^^^^^
    |
    = note: #[warn(stable_features)] on by default
```

But I can take care of this in a future PR.